src/control/lib/spdk/ctests/SConscript

```diff
@@ -0,0 +1,31 @@
+"""Build go-spdk bindings C tests"""
+from __future__ import print_function
```

Comment:
(pylint-misplaced-future) __future__ import is not the first non docstring statement
(pylint-reimported) Reimport 'print_function' (imported line 3)

Reply:
Are you still seeing this? I believe we do prints in other places in Scons files but maybe not

Reply:
would like to request an exception for this and the typedef checkpatch exceptions please on the grounds that they don't seem harmful?